pkg/controller/direct/privilegedaccessmanager/entitlement_controller.go

```diff
@@ -320,6 +320,8 @@ func (a *Adapter) Update(ctx context.Context, updateOp *directbase.UpdateOperati
 		log.V(2).Info("'spec.additionalNotificationTargets' field is updated (-old +new)", cmp.Diff(parsedActual.AdditionalNotificationTargets, parsedDesired.Spec.AdditionalNotificationTargets))
 		updateMask.Paths = append(updateMask.Paths, "additional_notification_targets")
 	}
+	defaultApprovalWorkflowManualApprovalsRequireApproverJustification(parsedActual)
```

Comment:
We should not change the actual value (then it is no longer "actual").   What happens if we remove ln323, will the test fail? In other words, does the GCP service give a different default in the actual value?